zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -65,7 +65,7 @@
 public class QuorumPeerConfig {
 
     private static final Logger LOG = LoggerFactory.getLogger(QuorumPeerConfig.class);
-    private static final int UNSET_SERVERID = -1;
+    public static final int UNSET_SERVERID = -1;
```

Comment:
I think `NOT_SET` sounds better, but if this is part of the original ZK code, then let's not change the name. I think changing the scope from `private` to `public` is okay since it won't break things.

Reply:
I'm reverting this change since we removed the old constructor.